utils/work/v1/workapplier/workapplier.go

```diff
@@ -121,7 +126,33 @@ func manifestsEqual(new, old []workapiv1.Manifest) bool {
 	}
 
 	for i := range new {
-		if !equality.Semantic.DeepEqual(new[i].Raw, old[i].Raw) {
+		if !equality.Semantic.DeepEqual(manifestUnstructured(new[i]), manifestUnstructured(old[i])) {
+			return false
+		}
+	}
+	return true
+}
+
+func manifestUnstructured(manifest workapiv1.Manifest) *unstructured.Unstructured {
+	newManifest := &unstructured.Unstructured{}
+	if err := newManifest.UnmarshalJSON(manifest.Raw); err != nil {
+		klog.Errorf("failed to unmarshal new manifest. err:%v", err)
+		return newManifest
+	}
+
+	unstructured.RemoveNestedField(newManifest.Object, "metadata", "creationTimestamp")
+	unstructured.RemoveNestedField(newManifest.Object, "spec", "template", "metadata", "creationTimestamp")
+
+	return newManifest
+}
+
+// mapExisted is used to check if required is included in existing.
+func mapExisted(required, existing map[string]string) bool {
```

Comment:
I think it is more like `shouldUpdateMap` ?

Reply:
yes, changed the func name.